e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -446,9 +447,14 @@ def _get_climo_dataset(self, season: str) -> xr.Dataset:
         if "slat" in ds.dims:
             ds = ds.drop_dims(["slat", "slon"])
 
-        all_vars = list(ds.data_vars.keys())
-        keep_bnds = [var for var in all_vars if "bnd" in var or "bounds" in var]
-        ds = ds[[self.var] + keep_bnds]
+        all_vars_keys = list(ds.data_vars.keys())
+        hybrid_var_keys = set(list(sum(HYBRID_SIGMA_KEYS.values(), ())))
+        keep_vars = [
+            var
+            for var in all_vars_keys
+            if "bnd" in var or "bounds" in var or var in hybrid_var_keys
+        ]
+        ds = ds[[self.var] + keep_vars]
```

Comment:
The build breaks in this PR because we didn't keep hybrid variables to reconstruct pressure from hybrid in PR #750. This should fix the build.